### ISSUE TITLE
Sugest to use many accounts

### ISSUE THREAD

**Issue by gpxlnx** [incogbyte/shosubgo#14]
What you think about use a list of shodan APIs ?

**Comment by incogbyte** [comment]
@gpxlnx u talk about rotation keys ?

**Comment by noob6t5** [comment]
@incogbyte  and  @gpxlnx  can you please check PR ,  I have  tried to update code  accordingly.

**Comment by gpxlnx** [comment]
> O que você acha de usar uma lista de APIs do Shodan?

Exactly, it would be really cool if shosubgo used a list of APIs to collect domains.

Enabling the use of several free APIs.

**Comment by gpxlnx** [comment]
> @incogbyte e @gpxlnx você pode verificar o PR, tentei atualizar o código adequadamente.

Great work, as soon as the PR goes up I will check and test it.

Thanks for the work.

**Comment by noob6t5** [comment]
Please take a look at it

**Comment by incogbyte** [comment]
i'll check the PR from @noob6t5 soon..
